src/components/Button/Button.stories.tsx

```diff
@@ -75,9 +75,7 @@ export const Link = {
 
   args: {
     children: "Link button",
-    appearance: "base",
-    element: "a",
-    href: "#test",
+    appearance: "link",
```

Comment:
Now looking at it, it got me thinking.

Maybe the intention was for the story to be about a link button (a link element), not about appearance.

So the existing example was likely correct. It was creating a button using `<a>` element instead of `button`.

But it is true, that because we have a "link" appearance it's quite confusing.


So maybe the fix is actually to add a new story about "link" appearance button, not to update the existing one.

It would be good to also add some kind of descriptions to the stories, so it's not so confusing what's the intention of the example.

Reply:
I think the existing one should be updated. All the examples around it are about the `appearance` property: default, base, positive, negative, brand - these are all possible appearance values, and the `link` appearance is the only one that is not represented by any example. The example with the `<a>` element was in the middle of them, so it looks out of place to me. Not sure why it was even added in the first place.

I can add back the previous example, with the `<a>` element, at the bottom of the list, if we really need to show such example, but for coherence with the examples around it, it should not be called "Link button". I'm open to suggestions for a better name, if we really cannot drop it altogether (which I still think would be the best choice here).

Reply:
Makes sense.

As for the `<a>` example I'd suggest to keep it (as you mentioned, makes sense to move it down). This is the only place where we seem to document the usage of `element` and `href` props. Maybe "Button as a link" for a title would be clear enough?

Reply:
Thanks for the suggestion. I think it's a good title. I've added the example back, plus the descriptions for all stories for `Button` [here](https://github.com/canonical/react-components/pull/1176/commits/23b9d1eab780c1b4df67436e8b11475dee7d1631).